frontend/components/FlashMessage/FlashMessage.tsx

```
@@ -94,11 +94,19 @@ const SingleFlashMessage = ({
     <div className={baseClasses} id={baseClasses}>
       <div className={`${baseClass}__content`}>
         <Icon
-          name={alertType === "success" ? "success" : "error"}
-          // Static (un-themed) so icon stays light on the colored toast
-          // surface in both light and dark mode.
+          name={
+            alertType === "success"
+              ? "success"
+              : alertType === "warning-filled"
+              ? "warning"
+              : "error"
```

ESLint

[error] Do not nest ternary expressions.

(no-nested-ternary)